report-modules-plugins.py

```diff
@@ -602,6 +602,7 @@ def process_role(role_path, is_real_role, ctx):
             ctx.found_role_name = get_role_name(role_path)
         ctx.enter_role(ctx.found_role_name, role_path)
         ctx.add_local_plugins(role_path, "library")
+        ctx.add_local_plugins(role_path, "tests/library")
```

Comment:
This should really be done in the `tests` context.  Consider this case:

Developer accidentally uses the plugin defined in `tests/library` in a `tasks/*.yml` file.  Then `report-modules-plugins.py` will pass, and the integration tests will pass, but the role will fail at runtime.